services/order.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict], username: str, date: datetime = None
+) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+            order.save()
+
+        [
+            Ticket.objects.create(
+                movie_session_id=ticket.get("movie_session"),
+                order=order,
+                row=ticket["row"],
+                seat=ticket["seat"]
+            )
+            for ticket in tickets
+        ]
+
+
+def get_orders(username: str = None) -> QuerySet:
+    queryset = Order.objects.all().order_by("-user")

```

Comment:
`.order_by()` is not needed here. The tests used to fail on some students' computers because of a bug that should now be fixed. The tests should pass on GitHub either way, even if they fail locally.